crates/bevy_internal/src/lib.rs

```diff
@@ -141,3 +138,31 @@ pub mod dynamic_plugin {
 
 #[cfg(target_os = "android")]
 pub use ndk_glue;
+
+/// `use bevy::prelude::*;` to import common components, bundles, and plugins.
+pub mod prelude {
+    #[doc(hidden)]
+    pub use crate::{
+        app::prelude::*, asset::prelude::*, core::prelude::*, ecs::prelude::*, log::prelude::*,
+        math::prelude::*, render::prelude::*, transform::prelude::*, window::prelude::*,
+        DefaultPlugins, MinimalPlugins,
+    };
+
+    pub use bevy_derive::bevy_main;
+}
+
+pub mod prelude2d {
```

Comment:
Other modules that end with `2d` and `3d` have an underscore prefix. These might be better as `prelude_2d` and `prelude_3d` to remain consistent with other parts of the API.

https://github.com/bevyengine/bevy/blob/adb3ad399cf092110b2d69f7dc189abc0ebc9f80/crates/bevy_core_pipeline/src/lib.rs#L3-L4